app/_kong_plugins/opa/index.md

```diff
@@ -32,6 +32,137 @@ categories:
+         "accept-encoding": "gzip, deflate",
+         "connection": "keep-alive",
+         "accept": "*\\/*"
+       },
+       "uri_captures": {      # The regex capture groups captured on the Kong Gateway Route's path field in the current request. Injected only if `include_uri_captures_in_opa_input` is set to `true`.
+         "named": {},
+         "unnamed": []
+       }
+     }
+   },
+   "client_ip": "127.0.0.1",# client IP address as interpreted by Kong
+   "service": {             # The Kong service resource to which this request will be forwarded to if OPA allows. Injected only if `include_service_in_opa_input` is set to `true`.
+     "host": "httpbin.konghq.com",
+     "created_at": 1612819937,
+     "connect_timeout": 60000,
+     "id": "e6fd8b19-89e5-44e6-8a2a-79e8bf3c31a5",
+     "protocol": "http",
+     "name": "foo",
+     "read_timeout": 60000,
+     "port": 80,
+     "updated_at": 1612819937,
+     "ws_id": "d6020dc4-67f5-4c62-8b45-e2f497c20f5c",
+     "retries": 5,
+     "write_timeout": 60000
+   },
+   "route": {               # The Kong route that was matched for this request. Injected only if `include_route_in_opa_input` is set to `true`.
+     "id": "bc6d8617-76a7-441f-aa40-32eb1f5be9e6",
+     "paths": [
+       "\\/"
+     ],
+     "protocols": [
+       "http",
+       "https"
+     ],
+     "strip_path": true,
+     "created_at": 1612819949,
+     "ws_id": "d6020dc4-67f5-4c62-8b45-e2f497c20f5c",
+     "request_buffering": true,
+     "updated_at": 1612819949,
+     "preserve_host": false,
+     "regex_priority": 0,
+     "response_buffering": true,
+     "https_redirect_status_code": 426,
+     "path_handling": "v0",
+     "service": {
+       "id": "e6fd8b19-89e5-44e6-8a2a-79e8bf3c31a5"
+     }
+   },
+   "consumer": {            # Kong consumer that was used for authentication for this request. Injected only if `include_consumer_in_opa_input` is set to `true`.
+     "id": "bc6d8617-76a7-431f-aa40-32eb1f5be7e6",
+     "username": "kong-consumer-username"
+   }
+ }
+}
+```
+
+You can use this structure, along with the [OPA docs](https://www.openpolicyagent.org/docs/latest/policy-language/), as a reference to create your policies. For example, you can use `input.request.http.querystring.page` to refer to a `page` query parameter.
+
+## OPA response
+
+Once OPA is done executing policies, the plugin expects the policy evaluation result as either a boolean or an object. If OPA returns any other format or a status code other than `200 OK`, the plugin will return a `500 Internal Server Error` to the client.
+
+### Boolean response
+
+OPA can return a `true` or `false` result after a policy evaluation. If the input request meets the defined policies, OPA should send the following response:
+```json
+{
+ "result": true
+}
+```
+
+If the request violates the policy, OPA should send the following response:
+```json
+{
+ "result": false
+}
+```
+In this case, any other fields in the response are ignored.
```

Comment:
```suggestion
OPA can return a `true` or `false` result after a policy evaluation. If the input request meets the defined policies, OPA should send a `"result": true` response.  If the request violates the policy, OPA should send a `"result": false` response. In this case, any other fields in the response are ignored.
```
Shortening this since there's not much in the JSON example.